pom.xml

```diff
@@ -135,6 +135,10 @@
         <thymeleaf.version>3.0.14.RELEASE</thymeleaf.version>
         <pmd.version>3.15.0</pmd.version>
         <checkstyle.version>3.1.2</checkstyle.version>
+        <sonar.organization>allbaa07</sonar.organization>
```

Comment:
We do not want these sonar properties inside the pom.xml file. If you want to use SonarQube you can use command line parameters for your build and test